CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 - server: few relay fixes (fix #5020, #5037, #5046) (#5013)
 - server: raise error on startup when `--unauthorized-role` is ignored (#4736)
 - server: fix bug which arises when renaming/dropping a column on a remote relationship (#5005, #5119)
+- server: unlock locked scheduled events on graceful shutdown
```

Comment:
I think you should remove this changelog. It is not user-facing.

Reply:
I think it’s worth keeping, for what it’s worth.

Reply:
@codingkarthik Can you move it under `Next release` ?